pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -423,6 +423,11 @@ type CommonPrometheusFields struct {
 	// Otherwise the HonorTimestamps field of the service or pod monitor applies.
 	OverrideHonorTimestamps bool `json:"overrideHonorTimestamps,omitempty"`
 
+	// When true, Prometheus ignores the timestamps for all the targets created
+	// from service and pod monitors.
+	// Otherwise the TrackTimestampsStaleness field of the service or pod monitor applies.
+	OverrideTrackTimestampsStaleness bool `json:"overrideTrackTimestampsStaleness,omitempty"`
```

Comment:
Quick question.
Do we see any use cases where Prometheus admin wants to override the decision made by the scrape config?

Reply:
When `OverrideTrackTimestampsStaleness` is set to "true" for Job target, it will be inserted in the TSDB when the target is down.

Reply:
@nicolastakashi @simonpasquier  Is it necessary to retain the `OverrideTrackTimestampsStaleness` global configuration?🤔

Reply:
I would keep it simple and drop it for now.

Reply:
From my understanding we can remove it for now and focus only on svc/pod monitor as @simonpasquier mentioned.

Reply:
Thanks @simonpasquier @nicolastakashi , PTAL
Drop the `OverrideTrackTimestampsStaleness` global configuration.